styles.css

```diff
@@ -455,6 +799,115 @@ label {
   border: none;
 }
 
+#ms-lb {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  .contact {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    background-image: url(./assests/images/image-geometry_6.png);
+    background-repeat: no-repeat;
+    background-position: left top;
+  }
+
+  .introduction-contact {
+    display: grid;
+    grid-template-columns: 20% 20% 20% 20% 20%;
+    grid-template-rows: auto auto;
+    grid-template-areas:
+      ". header header header ."
+      ". paragraph paragraph paragraph .";
+  }
+
+  .contact-header {
+    grid-area: header;
+    font-size: 90px;
+    text-align: center;
+    margin-bottom: 30px;
+  }
+
+  .contact-paragraph {
+    grid-area: paragraph;
+    font-size: 20px;
+    text-align: center;
+    padding-left: 5%;
+    padding-right: 5%;
+  }
+
+  .contact-form > ul {
+    display: grid;
+    grid-template-columns: 20% 30% 30% 20%;
+    grid-template-rows: auto auto auto auto;
+    grid-template-areas:
+      ". user email ."
+      ". message message . "
+      ". message message . "
+      ". button button .";
+    gap: 10px;
+  }
+
+  .user {
+    grid-area: user;
+  }
+
+  .email {
+    grid-area: email;
+  }
+
+  .message {
+    grid-area: message;
+  }
```

Comment:
- [ ] As requested by the previous reviewer, kindly ensure that your design matches the Figma template as much as possible. The contact form is too wide and the `name` and `email` text field height is small compare to the template. please fix this.

| Your design | Figma template |
| ------------- | ------------- |
| ![Capture2](https://user-images.githubusercontent.com/85095702/160215359-6021287f-f5ab-4ea4-ba8b-a9c29b4fb68d.PNG) | ![figma2](https://user-images.githubusercontent.com/85095702/160215393-6c6caafc-6c41-48c0-94f2-d8d7de17c069.PNG) |